api/controller/xtra_db_cluster.proto

```diff
@@ -86,5 +88,5 @@ message XtraDBClusterParams {
     }
   ];
   // Paused is true when cluster is suspended.
-  bool paused = 4;
+  bool paused = 4 [deprecated = true];
```

Comment:
and this boolean, because we haven't released it yet.

Reply:
Removed.